tf_sig_build_dockerfiles/devel.usertools/cpu.bazelrc

```diff
@@ -29,3 +29,6 @@ build --copt=-mavx --host_copt=-mavx
 
 # Use the NVCC toolchain to compile for manylinux2010
 build --crosstool_top=@ubuntu18.04-gcc7_manylinux2010-cuda11.2-cudnn8.1-tensorrt7.2_config_cuda//crosstool:toolchain
+
+# Save build profile.
```

Comment:
Can you move this to be in the common section (e.g. right after AVX in both files) and add something more to the comment? It doesn't currently add very much useful information for a confused reader. e.g. 

>Store performance profiling log in the mounted artifact directory. See https://docs.bazel.build/versions/main/skylark/performance.html#performance-profiling

Reply:
Done.